wiki/cpp-tutorial/language-basics.md

```diff
@@ -81,86 +81,90 @@ int foo = 0;   // declaration statement
-variable with an initial value.
+When we first create a variable in C++, it undergoes through a process called _initialization_. 
+Initialization means providing the variable with its initial value.
 
 ```cpp
 int foo = 0;
 ```
 
-The above statement is one of the most common ways to initialize a variable. It gives the variable `foo` an initial
-value of `0`. There are many ways to initialize a variable, each having its own pros and cons. The different types of
-initialization will be discussed in more detail in another section.
+The statement above is one of the most common ways to initialize a variable. Here, the variable `foo` is 
+initialized with the value of `0`. There are several ways to initialize a variable in C++, each with its own 
+pros and cons. These methods will be discussed in more detail in a later section.
 
-Assignment is a process that takes place after a variable already exists. It is the process of assigning a new value to
-an existing variable.
+Assignment, on another hand, occurs _after_ a variable has already been created. It refers to giving a new value 
+to an existing variable
 
 ```cpp
 int foo = 0; // initialization (begins with the type of the variable)
 foo = 2;     // assignment (no type specified)
 ```
 
-Like initialization, assignment has several different forms that will be discussed in more detail in a later chapter.
+Like initialization, assignment also comes in different forms, which will be explored 
+in more depth in a later chapter.
 
 ## Fundamental Data Types and Literals
 
-All values in C++ have a _type_, and variables can only hold values of a specified type.
+In C++, every values has a _type_, and variables can only store values of the type they are declared with.
 
-The most simple types are _fundamental types_. You have already seen variables of (fundamental) type `int` in the
-examples above, and values such as `0` (also of type `int`). We will go over some commonly used fundamental types in the
-following section.
+The most simple types are _fundamental types_. You have already seen examples of variables of (fundamental) 
+type `int`, and values such as `0`, which is also a type of `int`. 
+
+In this section, we will explore some commonly used fundamental types.
 
 ### `bool`
 
-The type `bool` can either hold the values `true` and `false`. You can use it to store the result of logic, like:
+The `bool` type reprsents the Boolean value, it can either hold `true` or `false`. You can use it to store the 
+result of logical expressions, like:
 
 ```cpp
-bool b = 1 > 0; // True if one is greater than zero.
+bool b = 1 > 0; // true, if 1 is greater than 0.
 ```
 
 ### Integer Types
 
-_Integer types_ in C++ are similar to integers in math. They can store (possibly negative) whole numbers like `1` or
-`-1`, but no fractions, like `0.5`.
+_Integer types_ in C++ are similar to whole numbers in mathematics. They can store values like `1`, `0` or `-1`,
+but nto fractions like 0.5.
```

Comment:
"whole numbers" seems weird. "Integer" is also the term used in mathematics; the original was fine.